src/shared/Core/Interop/Posix/PosixProcessManager.cs

```diff
@@ -0,0 +1,7 @@
+namespace GitCredentialManager.Interop.Posix;
+
+public class PosixProcessManager : ProcessManagerBase
+{
+    public PosixProcessManager(ITrace2 trace2) : base(trace2)
+    { }
+}
```

Comment:
If this doesn't `override` anything from `ProcessManagerBase`, why not just drop `PosixProcessManager` and just use `ProcessManagerBase` directly instead.

Reply:
Maybe rename `ProcessManagerBase` to `ProcessManager` too.